### PR TITLE
Submitting Strategy [gmc]

### PR DESCRIPTION
**Pull request by info-gokumarket**
Submitting a strategy to fetch the voting power

Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/strategies/gmc/index.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    ]);
+
+    callData = [...chunk(callData, options.max || 400)];
+    const response: any[] = [];
+    for (const call of callData) {
+      const multiRes = await multicall(network, provider, abi, call, {
+        blockTag
+      });
+      response.push(...multiRes);
+    }
```
Comment (ChaituVR) [inline src/strategies/gmc/index.ts:48]: Hi you can call multicall like this https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/index.ts#L26 to send all calls at once

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by info-gokumarket** [comment]
This strategy is only for user's wallet balance.

File: src/strategies/gmc/index.ts (hunk 1 above)
Comment (info-gokumarket) [reply src/strategies/gmc/index.ts:49]: Hi @ChaituVR  Tried it but its not working.

Comment (info-gokumarket) [reply src/strategies/gmc/index.ts:49]: when test is failing on your side its saying strategy not found.

**Comment by info-gokumarket** [comment]
> > This strategy is only for user's wallet balance.
> > If it is only for user's wallet balance you can use `erc20-balance-of` strategy
>> Voting power is equivalent to user's wallet balance

**Pull request closed without merging**